scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -316,6 +327,77 @@ task GetMaxTableId {
   }
 }
 
+task CheckForDuplicateData {
+    input{
+      String project_id
+      String dataset_name
+      Array[String] sample_names
+      File? service_account_json
+      # needed only for lockfile
+      String output_directory
+      String run_uuid
+      # runtime
+      Int? preemptible_tries
+  }
+
+  String has_service_account_file = if (defined(service_account_json)) then 'true' else 'false'
+   
+  
+  command <<<
+    set -e
+
+    if [ ~{has_service_account_file} = 'true' ]; then
+      gcloud auth activate-service-account --key-file='~{service_account_json}'
+      gcloud config set project ~{project_id}
```

Comment:
should we also set the environment variable here?  I don't know if it's needed but just to be consistent everywhere 

```
            export GOOGLE_APPLICATION_CREDENTIALS=~{service_account_json}
```

Reply:
we only need to set the environment variable when the gatk tool needs to access bigquery. Also, I believe (but have not verified) that when we set it, we have to localize all input files that are not in the aou project. Meaning that if we try to pass a gs path to the GATK that is in a workspace bucket, it won't be able to read the file.